content/en/upgrade/_index.md

```diff
@@ -1,4 +1,7 @@
 ---
+aliases:
+- /https://kb.altinity.com/altinity-kb-setup-and-maintenance/cluster-production-configuration-guide/version-upgrades/
+- /https://kb.altinity.com/altinity-kb-setup-and-maintenance/clickhouse-versions/
```

Comment:
```suggestion
```

Since that page will still be avaliable and have a link to the proper one - that's seem not needed.